static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -326,7 +326,9 @@ define([
                         resolve();
                     }
 
-                    if (abVideoAutoplay.variant === 'autoplay') {
+                    var contentType = config.page.contentType;
+                    var isArticleOrLiveBlog = contentType === 'Article' || contentType === 'LiveBlog';
+                    if (abVideoAutoplay.variant === 'autoplay' && isArticleOrLiveBlog) {
```

Comment:
Strangely I would prefer not to do this as this also requires the switch to be on - which I don't want to do until I have run it past a few people. Having this would stop the `#ab-TESTNAME` from working.